lib/asset_sync/config.rb

```diff
@@ -175,6 +174,18 @@ def public_path
       @public_path || ::Rails.public_path
     end
 
+    def public_path=(path)
+      # Generate absolute path even when relative path passed in
+      # Required for generating relative sprockets manifest path
+      @public_path = if path.absolute?
```

Comment:
I would have actually expected this to be a string relative path `'./public'` if we are setting it in an initializer.

I tested in my console, and Pathname seems to be idempotent. So we should probably pass path into Pathname so we can call absolute on it, or it will blow up on a regular string.
The code below returns a Pathname instance, which has the `.absolute` method defined.
```suggestion
pathname = Pathname.new(path)
@public_path = if path.absolute?
```

Tested in rails console:
```ruby
Pathname.new(Pathname.new('./public'))
# => #<Pathname:./public>
```

I assume step 2 would be to add this to the config options listed [here](https://github.com/AssetSync/asset_sync#custom-rails-initializer-configinitializersasset_syncrb). A comment to know this can handle Pathname instances or a relative path from project root would be helpful as well.

Reply:
Yup I missed that since the spec used `Pathname` in setup
Thanks for reminding me that

Updated code, spec setup and doc